### PR TITLE
Added Chargebee Integration docs

### PR DESCRIPTION
**Pull request by williamhatcher**
And fixed a few typos in the FiveM docs

### CODE REVIEW

File: src/integrations/fivem.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ You will then be prompted for your *FiveM* server ID. This is the 6 character al
 The integration is now active in your server! The last step is to insert the placeholders of your choice into the ticket welcome message. To do this, head over to the "Reaction Panels" page, press "Edit" on a ticket panel, expand advanced settings, and open the welcome message editor:
 ![Welcome message editor](/img/integrations/edit_welcome_message.webp)
 
-Then, customise the welcome message to your liking:
-![Welcome message editor](/img/integrations/placeholders.webp)
+Then, customize the welcome message to your liking:
```
Comment (rxdn) [inline src/integrations/fivem.md]: British English!

**Review by rxdn — CHANGES_REQUESTED** [review]
Two things:
- Could you remove the `src/img/integrations/chargebee/.DS_STORE` file (Add `.DS_Store` to the gitignore file if you're feeling nice)
- You have added the images as PNGs, these need to be converted to webp

File: src/integrations/fivem.md (hunk 1 above)
Comment (williamhatcher) [reply src/integrations/fivem.md]: 😃

**Comment by williamhatcher** [comment]
@rxdn Issues fixed. I don't have Docker to easily preview this on my local machine though.

**Comment by rxdn** [comment]
> @rxdn Issues fixed. I don't have Docker to easily preview this on my local machine though.

~~Looks good, could you change `customize` back to `customise` though, and then I'll merge~~
Ignore, I did it :P

**Comment by rxdn** [comment]
If you have cargo installed on your machine, you can `cargo install mdbook` and then `mdbook serve`

**Comment by williamhatcher** [comment]
I’ll preview it soon.

My editor was sternly yelling at me for misspelling “customise” when I copied the fivem doc. Apologies. I’ll change it in a bit.
________________________________

**Pull request merged**